WooCommerce/WooCommerce.xcodeproj/project.pbxproj

```diff
@@ -4401,6 +4405,7 @@
 				3FF314E026FC74450012E68E /* UITestsFoundation.h */,
 				F997173223DBCF2800592D8E /* XCTest+Extensions.swift */,
 				80C3626A27704EE1005CEAD3 /* ProductDataStructs.swift */,
+				80C3627027745737005CEAD3 /* ReviewDataStructs.swift */,
```

Comment:
I'm glad to see you put this new data transfer object in the right place™, meaning within the UITestsFoundation internal framework. Well done 👍